pages/common/prosodyctl.md

```diff
@@ -0,0 +1,23 @@
+# prosodyctl
+
+> The control tool for the Prosody XMPP server.
+
+- Show the status of the Prosody server:
+
+`sudo prosodyctl status`
+
+- Reload the server's configuration files:
+
+`sudo prosodyctl reload`
+
+- Add a user to the Prosody XMPP server:
+
+`sudo prosodyctl adduser user@example.com`
+
+- Set a user's password:
+
+`sudo prosodyctl passwd user@example.com`
+
+- Permanently delete a user:
```

Comment:
Yes, it is the default behavior that deleting data is a permanent operation, especially in the command line. The "recycling" behavior is an exception and normally reserved for GUI environments.

And while I understand that it's better to be explicit, in this case it may actually raise doubts about whether non-permanent deletion is available.

That said, IMO the balance between these concerns is thin, so I'm ok with either decision.